main/modes/games/cosCrunch/cosCrunch.c

```diff
@@ -37,6 +37,36 @@ static const char* cosCrunchHowToPlayText[]
        "See how many crafts you can complete before MAGFest arrives. Your score depends on it! You'll never really be "
        "done, but maybe you can get close enough."};
 
+#define CC_NVS_NAMESPACE      "cc"
+#define NVS_KEY_TUTORIAL_SEEN "tutorialSeen"
```

Comment:
C tip, it's a bit better to declare these as `const char[] foo = "bar";` because there will be only one variable in compiled code, whereas each copy of this `#define` will be another string in ROM. We're not that desperate for bytes, but food for thought.